### PR TITLE
Added UserID to the user get list

### PR DESCRIPTION
**Pull request by karl-police**
Added UserID to the user get list
these days we need UserID's too. To use it directly with other bots that only support UserID's, very useful to get bots or get their userid very fast.

If you think, you can use the user discrimination with blargbot, yeah but, probably not sure haven't tested it but I use another bot that logs infractions and it only supports UserID's. Also this is not bad to list UserID's to directly report them.

Please add it

https://airtable.com/shrEUdEv4NM04Wi7O/tblyFuWE6fEAbaOfo/viwDg5WovcwMA9NIL/recXjAjG3xzMaQbmd?blocks=bipip3uBZFRPcSy3C

should this be added, then please build it

### CODE REVIEW

**Comment by JonasDanmark18** [comment]
this is very nice! I can see the potential of this feature! It would help a lot - great work 👍.

File: src/utils/generic.js [hunk 1]
```diff
@@ -606,7 +606,7 @@ bu.getUser = async function (msg, name, args = {}) {
                 newUserList.push(userList[i]);
             }
             for (let i = 0; i < newUserList.length; i++) {
-                userListString += `${i + 1 < 10 ? ' ' + (i + 1) : i + 1}. ${newUserList[i].user.username}#${newUserList[i].user.discriminator}\n`;
+                userListString += `${i + 1 < 10 ? ' ' + (i + 1) : i + 1}. ${newUserList[i].user.username}#${newUserList[i].user.discriminator} - ${newUserList[i].user.id}\n`;
```
Comment (karl-police) [inline src/utils/generic.js]: Added so that it shows the UserID too

File: src/utils/generic.js [hunk 2]
```diff
@@ -606,7 +606,7 @@ bu.getUser = async function (msg, name, args = {}) {
...
             }
             let moreUserString = newUserList.length < userList.length ? `...and ${userList.length - newUserList.length}more.\n` : '';
             try {
```
Comment (karl-police) [inline src/utils/generic.js]: test

**Comment by Sergeant22** [comment]
I hope the creator actually see that

**Comment by karl-police** [comment]
Merge :)

**Comment by danny-may** [comment]
Only @Ratismal can merge PRs and deploy them im afraid.

**Pull request merged**